### DISCUSSION TITLE
How does performance compare to timesfm?

### DISCUSSION THREAD

**Discussion by nxfi777** [ztxtech/Time-Evidence-Fusion-Network#2] [Q&A] [answered]
Fantastic paper, but it would be great to see some comparisons to google's decoder-only foundation model, [timesfm](https://github.com/google-research/timesfm/)

**Comment by ztxtech** [comment, marked as answer]
Wow. TimesFM looks like a new work and is being published in ICML 2024. I roughly browsed through it and it seems that TimesFM is a pre-trained model, and relevant data can be compared, such as the ETT dataset. Actually, this comparison is not  fair enough because of pre-training, lol.

**Reply by nxfi777** [reply]
Yeah I was looking for matching benchmarks but couldn't find. Thanks anyway, I will try this.

**Reply by ztxtech** [reply]
You are welcome. There is a recommended open source benchmark [TSlib](https://github.com/thuml/Time-Series-Library), and developing of TEFN is based on TSlib. Maybe it is helpful, we can wait for the official scripts of TimesFM.
